qubesmanager/settings.py

```diff
@@ -401,6 +408,23 @@ def check_mem_changes(self):
                         "allowed value."))
             self.init_mem.setValue(self.max_mem_size.value() / 10)
 
+    def _run_in_thread(self, func, *args):
```

Comment:
Note for the future: better put refactoring (without functional change) into a separate commit.